test/src/core/testPageObjectWaitForElementNotPresent.js

```diff
@@ -18,16 +18,17 @@ describe('test PageObject WaitForElementNotPresent', function () {
     Nocks.deleteSession().disable();
   });
 
-  it('WaitForElementNotPresent with section', function(done) {
+  it('WaitForElementNotPresent with section', async function() {
     Nocks.elementFound().click().elementFound().childElementsFound('#badElement').elementFound().childElementsNotFound()
 
     const page = this.client.api.page.waitForElementNotPresentPageObj();
 
+    let res;
     page.waitForElementNotPresentDemo(function(result) {
-      assert.equal(result.status, 0);
-      done();
+      res = result;
     })
-
-    this.client.start()
+  
+    await this.client.start();
+    assert.equal(res.status, 0);
```

Comment:
We can probably do it like this:
```suggestion
   this.client.start(function (err) {
     if (err) {
       done(err);
       
       return;
     }
     
     try {
       assert.strictEqual(res.status, 0);
       done();
     } catch (err) {
       done(err);
     }
   });
```

Reply:
I might me wrong, but how is this code different from the `await` implementations?

Reply:
I'm not sure, maybe it's the same. It's just that it's how the tests are written at most places in the codebase, so just to be consistent. I didn't find any place where the assertion is done after the call to `client.start()`, but then `client.start()` is also not used with `await` anywhere. Also, [here](https://github.com/nightwatchjs/nightwatch/pull/3220#discussion_r890877667) Andrei had suggested that we should put the assertions inside a callback to `client.start()`.

Reply:
Andrei is right, to suggest then assertions should not exist there, because it breaks the tests in case of failure. But instead of using callbacks, I think it is much cleaner if we use the `await` function. Can you look at this once again and see if there are any logical differences? I am quiet confident that these are same, but I think someone else should also have a look just to be sure.

Reply:
@gravityvi maybe you can take a look?

Reply:
yes, it is the same.

Reply:
cool